Updated Inelastic Data Analysis Documentation

sf1919 opened the pull request:
**Description of work.**
Added an updated screenshot of the Ineleastic Data Analysis GUI to the Inelastic Data Analysis Documentation and the Elwin and I(Q,t) documentation. Also updated the Elwin documentation with details of the new loading facility including a gif demonstrating how to use.

**To test:**
- remove build/docs/doctrees
- build the documents again and check there are no warnings
- Check the information is still correct
- Are the images visible and clearly illustrate the new feature/improvement?
- Grammar/Spelling check


Fixes #32364

---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there is GUI work does it follow the [GUI standards](http://developer.mantidproject.org/Standards/GUIStandards.html)?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.

sf1919 committed:
Updated Inelastic Data Analysis Documentation

Added updated screenshot of GUI to Ineaslastic Data Analysis Documentation

sf1919 committed:
Updated Elwin documentation

Added two new screenshots to Elwin documentation and details of File/Workspace dropdown menu. Added release notes.

DanielMurphy22 commented on docs/source/interfaces/indirect/Indirect Elwin & Iqt.rst:
```diff
@@ -18,13 +18,19 @@ Mantid data search paths that contains the sample environment information the
 ~~~~~~~~~~~~~
 
+File or Workspace
+  Choose to download data from a file or a workspace by using this dropdown menu. See image below for demonstration of how to load files using either option.
```
Is `download` the correct word here? I would have thought just 'load' was better but I'm just checking!

sf1919 replied:
You're right it should be load!

DanielMurphy22 commented on docs/source/interfaces/indirect/Indirect Elwin & Iqt.rst:
```diff
@@ -18,13 +18,19 @@ Mantid data search paths that contains the sample environment information the
 of input files.
 
-.. interface:: Data Analysis
+.. image:: ../../images/Inelastic_Data_Analysis_GUI.png
   :width: 450
```
```suggestion
.. image:: ../../images/Inelastic_Data_Analysis_GUI.png
  :width: 750
  :align: center
```
Can we make this image larger and maybe centred?

DanielMurphy22 commented on docs/source/interfaces/indirect/Inelastic Data Analysis.rst:
```diff
@@ -23,7 +23,7 @@ The other two tabs in the interface perform transformations on data and are docu
 
-.. interface:: Data Analysis
+.. image:: ../../images/Inelastic_Data_Analysis_GUI.png
   :width: 650
 
```
Maybe you want the width and align parameters to be the same on both pages. 650 width I'd say is good as a minimum for this image, I like them large!

DanielMurphy22 reviewed (changes requested):
Really good to have these new images and gif! Just a few small things.

sf1919 committed:
Minor changes after review

Resized and centred screenshots. Changed download to load in Elwin load description.

DanielMurphy22 reviewed (approved):
Well done, this looks great! The gif is really impressive!
